mw-webapp/src/component/input/Input.tsx

```diff
@@ -74,6 +74,12 @@ interface InputProps<T extends string | number> {
    */
   parser?: (value: string) => T;
 
+  /**
+   * Style for input
+   */
+  style?: {
+    [key: string]: string;
+  };
 }
```

Comment:
I think it's not the best idea to use additional props style. Let's discuss about it in the group? Can you please write there advantages and disadvantages of this method? )

Reply:
I removed props style and did as you advised